docker-compose/technical/docker-compose.technical.yml

```diff
@@ -110,3 +110,31 @@ services:
       - PGADMIN_DEFAULT_PASSWORD=admin
     volumes:
       - $PWD/../technical/servers_pgadmin.json:/pgadmin4/servers.json
+
+  prometheus:
+    profiles:
+      - all
+      - metrics
+    image: prom/prometheus
+    command:
+      - --enable-feature=exemplar-storage
+      - --config.file=/etc/prometheus/prometheus.yml
+    volumes:
+      - $PWD/../technical/prometheus.yml:/etc/prometheus/prometheus.yml:ro
+    ports:
+      - "8000:9090"
+
+  grafana:
+    profiles:
+      - all
+      - metrics
+    image: grafana/grafana
+    volumes:
+      - $PWD/../technical/grafana/config/datasources:/etc/grafana/provisioning/datasources:ro
+      - $PWD/../technical/grafana/config/dashboards:/etc/grafana/provisioning/dashboards:ro
```

Comment:
set to read-only would block someone to modify/play with dashboard if it need it no?  
also have `editable: true` in _dashboard.yml_

Reply:
No it's to not modify the config files